content/documentation.rst

```diff
@@ -0,0 +1,370 @@
+-------------------------------------
+
+Examples of configuration or logs in the documentation should be edited to be
+more of a real world example. 
+
+For example, instead of 'user', add a real name such as 'alice' or 'bob':
+
+.. sourcecode:: rst
+
+    [accounts/mark-uuid]
+    name = mark
+    enabled = Yes
+    type = application
+    group = Staff
+    description = Staff SFTPPlus application account for Mark
+    home_folder_path = /PATH/TO/MARK/HOME
+    password = PASSWORD
+
+
+Updating the documentation
+--------------------------
+
+The release process will be similar to the rest of the software.
+
+An empty ``.ignore`` with the internal ID can be added in the release notes.
+
+Further details about generating and building documentation is found in the
+chevah server repository.
+
+When creating a new page, ensure to add the page name in the index.rst so that
+the page appears in the index documentation tree.
+
+
+Docstrings
+==========
+
+* Well documented code is extremely important.
+  Take time to describe components, how they work, their limitations, and the
+  way they are constructed.
+  Don't leave others in the team guessing what is the purpose of uncommon or
+  non-obvious code.
+
+* Document code as part of docstrings and not as comments.
+
+.. sourcecode:: python
+
+    def iamanExample(doc):
+        """
+        A simple docstring is placed here.
+        """
+          config = self.createSomethingHere('')
+
+There are two types of docstrings for production and test code.
+
+
+Test code docstrings
+--------------------
+
+Test code docstrings can contain information during the review process of new
+tests that can be written.
+
+.. sourcecode:: python
+
+    class TestHelpers(IAmATestCase):
+        """
+        The docstring here may add tests for helpers for a certain module
+        """
+        def test_of_a_module_1(self):
+         """
+         What is expected to happen in the first module of this test case
+         """
+        def test_of_a_module_2(self):
+         """
+         What is expected to happen in the second module of this first case
+         """
+
+
+Production code docstrings
+--------------------------
+
+Docstring are added in the production code to provide further information for
+readers and reviewers.  
+
+For example:
+
+.. sourcecode:: python
+
+    def getSomethingNewHere(self):
+
+Whereas a docstring should be added to add further information:
+
+.. sourcecode:: python
+
+    def getSomethingNewHere(self):
+        """
+        A docstring describing what SomethingNewHere is about
+        """
+
+
+Marketing / Promotional Materials
+=================================
+
+Promotions and marketing materials are located in the main website.
+
+It should be as generic and non-technical as possible with links to the
+Documentation for more in-depth / technical information.
+
+Please go to the internal wiki under General > Marketing for internal marketing
```

Comment:
Yes, I was undecided if I should even mention the internal wiki/Trac on the Styleguide..
I will take this out.